visualization/plot_measurement.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def plot_measurementdata(DataFilePath, ConditionFilePath):
```

Comment:
variable names should be lower case in python (https://www.python.org/dev/peps/pep-0008/)